receiver/mongodbreceiver/integration_test.go

```diff
@@ -137,42 +144,46 @@ func TestMongodbIntegration(t *testing.T) {
 	}
 
 	for _, tt := range testCases {
-		t.Run(tt.name, func(t *testing.T) {
```

Comment:
We should be able to pass in the func like this: `t.Run(tt.name, t.run)`

Reply:
updated